### PR TITLE
merging google and onboarding availability

### PR DESCRIPTION
**Pull request by WhoaNellie**

### CODE REVIEW

File: server/routes/googleCalendarAPI.js [hunk 1]
```diff
@@ -6,16 +6,55 @@ const router = new express.Router();
...
 router.post("/api/calendar/availability", async (req, res) => {
-  const day = req.query.day;
-  console.log(day);
+  const day = new Date(req.query.day);
   if (req.body.members.length > 1) {
     // team availability
   } else {
     try {
+      const availability = [];
       const usr = await db.User.findById(req.body.members[0]);
```
Comment (sina-jamshidi) [inline server/routes/googleCalendarAPI.js]: this is a small thing, but I much prefer having words printed out clearly for easier reading. Eg. `usr` vs `user`

File: server/routes/googleCalendarAPI.js [hunk 2]
```diff
@@ -6,16 +6,55 @@ const router = new express.Router();
...
-      const tokenStore = await db.AuthStore.findOne({ email: usr.email });
-      const authToken = tokenStore.googleAuthToken;
-      const availability = await getAvailability(authToken, day);
+      if (usr.availableDays.includes(day.getDay())) {
+        const tokenStore = await db.AuthStore.findOne({ email: usr.email });
+        const authToken = tokenStore.googleAuthToken;
+        const googleCal = await getAvailability(authToken, day);
```
Comment (sina-jamshidi) [inline server/routes/googleCalendarAPI.js]: same here. `googleCalendar`

**Review by sina-jamshidi — APPROVED** [review]
Nicely handled! I just added a comment enforcing my personal code style preference lol

File: server/routes/googleCalendarAPI.js (hunk 2 above)
Comment (WhoaNellie) [reply server/routes/googleCalendarAPI.js]: Can do!

**Pull request merged**